src/controllers/accommodation.controller.js

```diff
@@ -42,6 +45,27 @@ class AccommodationController {
     ResponseService.setSuccess(200, 'list of available accommodations', availableAccommodations);
     return ResponseService.send(res);
   }
+
+  /**
+       *
+       *
+       * @static
+       * @param {req} req
+       * @param {res} res
+       * @returns {response} @memberof AccommodationController
+       */
+  static async likeOrUnlikeAccommodation(req, res) {
+    const { accommodationId } = req.params, userId = req.userData.id;
```

Comment:
In order to save memory or not consume enough memory could pass `accommodationId` and `userId` inside `updateAccommodationLike` function. for example updateAccommodationLike({ accommodationId:  req.params.accommodationId}). 

What do you think about this?

Reply:
Ok. I implemented it @erickyvand